src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodDriver.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodDriver : GpioDriver
+    {
+        private const string GpioGet = "gpioget";
+        private const string GpioChip0 = "gpiochip0 ";
+        private const string GpioSet = "gpioset";
+        public int _waitMilliSeconds { get; set;}
```

Comment:
When using public properties we usually use CamelCasing. Also, people won't be able to configure this at all, because they call this method through the controller, which won't expose this property.